contracts/helpers.cairo

```diff
@@ -141,11 +141,11 @@ func _get_value_of_position{
     syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr
 }(
     option: Option,
-    position_size: felt,
-    option_type: felt,
-    current_volatility: felt,
-    current_pool_balance: felt
-) -> (position_value: felt){
+    position_size: Math64x61_,
+    option_type: OptionType,
+    current_volatility: Math64x61_,
```

Comment:
Could we differentiate between vol as percentage(100s) and decimal(1s)? Since we store hundreds but use decimals for calculations

Reply:
Perhaps an extra type for volatility would make sense, then? Do we use percentages elsewhere than in volatility, except for the instestines of black scholes?

Reply:
Definitely a good thing to separate types by what they actually store and not just the data format. Where is/would be VolDec used and what do you mean by it?

Reply:
afaik VolDec is an input for bs/pricing, and VolPerc is what is actually stored and every time it is used it's divided by fromFelt(100)

Reply:
I just want to make clear that the storage var stores vol as %, but some functions need it as decimal and make it clear what is returned by some getters/view funcs that read volatility

Reply:
I don’t get why would we work with volatility as a
decimal, it is inherently a percentage

Reply:
Ah I see, some functions take it as 100% = 1 and some 100% = 100. Looks like we should get rid of this, I’ll add this as a low prio task and replace Math64x61_ with a new Volatility type that should eventually be only 100% = 1.

Reply:
Sorry, I guess I didn't explain myself clear. It is as you've just said.